demo-forms/simprints-demo.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0"?>
```

Comment:
Deliberately here actually, so test forms for specific features can be found easily.  I understand that forms in general are in projects, but to me it doesn't really make sense for forms which demonstrate specific features or exercise specific enketo widgets.  I guess these forms are good for two uses:

* let developers easily upload a form to see a specific widget or enketo feature in action
* provide a solid example for tech leads building forms

However, the latter is probably less relevant as form XML is usually generated from XLSX.

I did the same on another branch for demonstrating mobile photo/direct-from-camera upload and again it seemed natural to have the example form in the `medic-webapp` repo.

Ultimately I don't mind though; let me know your thoughts.